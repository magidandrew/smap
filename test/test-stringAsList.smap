int main(){
    string song = ['y','o','\n','h','i','8'];
    list char duet = "yodelayheehoo~~~!";
    string melody = "birthday";
    duet[4] = 'O';    
    melody[4] = '8';
    println(melody);
    println(melody[0]);
    println(duet);
    print(song);
    return(0);
}