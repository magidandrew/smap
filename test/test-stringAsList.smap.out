birt8day
b
yodeOayheehoo~~~!
yo
hi8